openmmapi/include/openmm/internal/vectorize8.h

```diff
@@ -51,6 +51,19 @@ class fvec8 {
     fvec8(float v1, float v2, float v3, float v4, float v5, float v6, float v7, float v8) : val(_mm256_set_ps(v8, v7, v6, v5, v4, v3, v2, v1)) {}
     fvec8(__m256 v) : val(v) {}
     fvec8(const float* v) : val(_mm256_loadu_ps(v)) {}
+
+    /** Create a vector by gathering individual indexes of data from a table. Element i of the vector will
+     * be loaded from table[idx[i]].
+     * @param table The table from which to do a lookup.
+     * @param indexes The indexes to gather.
+     */
+    fvec8(const float* table, const int idx[8])
+    {
```

Comment:
Still some incorrect formatting here.  Opening brace should be at the end of the line.  Also in various other places below.

Reply:
Done